### PR TITLE
gates : draw direction gizmo on selected gate

### PR DESCRIPTION
**Pull request by Vroonsh**
Should close https://github.com/freezy/VisualPinball.Engine/issues/106

### CODE REVIEW

File: VisualPinball.Unity/VisualPinball.Unity.Editor/Inspectors/GateInspector.cs [hunk 1]
```diff
@@ -31,12 +32,42 @@ public class GateInspector : ItemInspector
...
+					Handles.color = Color.white;
+					Handles.DrawWireDisc(position, axis, scale);
+					Color col = Color.grey;
+					col.a = 0.25f;
+					Handles.color = col;
+					Handles.DrawSolidDisc(position, axis, scale);
+
+					Handles.color = Color.white;
+					var ratioscale = 2.0f;
+					var arrowscale = (worldScale * (100.0f - ratioscale)) + Mathf.PingPong(Time.realtimeSinceStartup * worldScale * ratioscale * 2.0f, worldScale * ratioscale);
```
Comment (freezy) [inline VisualPinball.Unity/VisualPinball.Unity.Editor/Inspectors/GateInspector.cs:62]: Can we get rid of the unused animation code?

**Review by freezy — APPROVED** [review]
Looking good, thanks again! :)

Can you clean up the animation code before I merge?

File: VisualPinball.Unity/VisualPinball.Unity.Editor/Inspectors/GateInspector.cs (hunk 1 above)
Comment (Vroonsh) [reply VisualPinball.Unity/VisualPinball.Unity.Editor/Inspectors/GateInspector.cs:62]: done

Comment (freezy) [reply VisualPinball.Unity/VisualPinball.Unity.Editor/Inspectors/GateInspector.cs:62]: With that it still animates, but only when the scene view camera moves, no?

Comment (freezy) [reply VisualPinball.Unity/VisualPinball.Unity.Editor/Inspectors/GateInspector.cs:62]: Hey @Vroonsh I think you still need to remove this one, then I'll merge!

Comment (Vroonsh) [reply VisualPinball.Unity/VisualPinball.Unity.Editor/Inspectors/GateInspector.cs:62]: > With that it still animates, but only when the scene view camera moves, no?

yup exactly

Comment (Vroonsh) [reply VisualPinball.Unity/VisualPinball.Unity.Editor/Inspectors/GateInspector.cs]: animation code removed for gates

**Pull request merged**